etl/embedding_model_pipelines/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .embedding_model_pipeline import EmbeddingModelPipeline as EmbeddingModelPipeline
```

Comment:
Nit: I'd probably have a single `.pipelines` module instead of `embedding_model_pipelines` and `generative_model_pipelines`.

Reply:
The module name has been renamed to `.pipelines`.